ui/webui/src/components/AnacondaAboutModal.jsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState, useEffect } from "react";
+import {
+    AboutModal,
+    Button,
+    Flex,
+    TextContent,
+    TextList,
+    TextListItem,
+} from "@patternfly/react-core";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons";
+import { getAnacondaVersion, getOsReleaseByKey } from "../helpers/product";
+
+export const AnacondaAboutModal = ({ isModalOpen, setIsAboutModalOpen }) => {
+    const toggleModal = () => {
+        setIsAboutModalOpen(!isModalOpen);
+    };
+
+    const [anacondaVersion, setAnacondaVersion] = useState("");
+    const [prettyName, setPrettyName] = useState("");
+
+    useEffect(() => {
+        getAnacondaVersion().then(content =>
+            setAnacondaVersion(content.split(" ").slice(-1)[0].replace("\n", ""))
+        );
+
+        getOsReleaseByKey("PRETTY_NAME").then(
+            setPrettyName
+        );
+    }, []);
+
+    return (
+        <>
+            <AboutModal
+              style={{ marginTop: "0" }}
+              isOpen={isModalOpen}
+              onClose={toggleModal}
+              trademark="Anaconda © 2023 Red Hat, Inc."
+              productName={prettyName}
+            >
+                <TextContent className="pf-u-py-xl" style={{ marginBottom: "48px" }}>
```

Comment:
Please dont use inline styles. Prefer using patternfly layout components for spacing, and if you still need to overwrite the default patternfly styles add a class and introduce the styles in a seperate scss/css file.